scripts/devel/build_and_install.py

```diff
@@ -77,7 +83,16 @@ def main(argv=sys.argv[1:]):
         if args.clean_after:
             clean_workspace(args.workspace_root)
 
-    return rc
+    if not args.abichecking:
+        return rc
+
+    with Scope('SUBSECTION', 'use abi checker'):
+        rc = call_abi_checker([args.workspace_root], args.rosdistro_name, env=env)
+
+    if rc != 0:
+        print("Failure during the execution of abi-checking")
+        return rc
+
```

Comment:
Nitpick: remove extra empty line.

Reply:
2ab7c8d

Reply:
Looks like 2ab7c8d was a different empty line?

Reply:
I think it was right. Note that code has changed a lot since that comment. Anyway I don't think that we have the problem anymore.